bin/xbundle_convert.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+"""
+A command-line script to use xbundle.
+"""
+
+import sys
+
+from xbundle import XBundle, RunTests
+
+# pylint: disable=star-args
+
+def usage():
```

Comment:
Yeah, I totally agree. I was going to put it into the TODO, but there's a bunch of stuff I plan to do after this pull request.